BOUNTY.md

```diff
@@ -1,24 +1,33 @@
+Issues that have a bounty are marked with two labels. Firstly, a cyan bounty label denotes that an issue has a bounty reward associated with it. Secondly, the amount of the reward in $MUSIC. Should you find that one of the bounty issues falls into your wheelhouse; by all means, submit a PR!
 
-Some general guidelines towards the same are:
+## Contributing
 
-1. The best proposal wins, not the first one. So you'll have to make sure that your fix stands out from the rest.
-2. The Musicoin developers would be happy to assist any person with a given issue, at the official Slack/Discord forums.
-3. To claim your bounty, you're required to sign the Musicoin CLA. This will be presented to you after your PR is merged.
-4. New bounty issues can be submitted at any time, the team will review and assign a bounty if needed.
-5. Teamwork is encouraged. If two people work together to fix an issue, both will be rewarded in proportion to their contribution.
-6. General guidelines like DRY and comments are strongly encouraged.
+1. Quality is preferred over turnover time; naturally. So put some effort in, and make your patch awesome.
+2. We would be more than happy to lend assistance if need be. You can find us at the official Slack or Discord.
+3. To claim your bounty, you're required to sign the Musicoin CLA. This will be presented to you if your PR is merged.
+4. New issues can be submitted at any time, the team will review and assign a bounty if needed.
+5. Teamwork is encouraged. If two people work together to fix an issue, both will be rewarded in proportion to their contribution. So pool your skills, and maybe you'll come up with something we didn't.
+6. Comment and fully document your work! This isn't to impose more hoops for you to jump though, but is in the interest of the others who may code on that file next. Make sure to attempt a solid explanation of anything that is not reasonably intuitive.
 
-The bounty program is open for all, beginner or experienced - Beginner issues will be labelled with a "good first issue" tag and assigned a small amount of bounty. The bounty amount may increase depending on the difficulty and urgency of the issue presented. All final calls regarding awarding bounty strictly lie within the Musicoin team and will not be subject to discussion.
+The bounty program is open for all, beginner or experienced. Issues that we think could be a good task for somebody first starting out are labelled a "good first issue", with an appropriate bounty. The awarded amount may increase depending on the difficulty and urgency of the issue presented. All decisions regarding bounties fall solely to the discression of the Musicoin core team.
```

Comment:
s/discression/discretion

Reply:
`for somebody first starting out` is grammatically wrong too. Could be simply "Issues that we think are good tasks for beginners are labelled as..."

Reply:
Rarely am I called out pedantically. Very, very rarely. Good call.